sips/sip-16/sip-016-token-metadata.md

```diff
@@ -0,0 +1,394 @@
+
+## Localization
+
+The localized data follow the same JSON schema with property `sip` as required and all other properties as optional.
+
+The localized data overwrite data provided in the default meta data JSON. The localized data can provide only partial data.
+
+An array of localized `attributes` overwrites the whole list of default `attributes`.
+
+A localized properties with partial data overwrites only the provided properties, the remaining default properties remain as default values.
+
+# Using metadata in applications
+
+Before presenting metadata to users, application developers should verify whether the metadata is compliant with the application's guidelines.
+
+We remind implementation authors that the empty string for the token uri is a valid response. We also remind everyone that any smart contract can use the same metadata as other NFT contracts. It is out of the scope of this standard to define how a client may determine which smart contracts are the original, well-known, canonical ones.
+
+## Graphical representation
+
+The metadata of a token contain several properties that can be used to visually represent the token. It is recommended to use the first defined property of the following list:
+
+1. `image`
+2. `properties.image_data`
+3. `image_data`
+
+Rich representation should use ther first defined property of the following list:
+
+1. `properties.animation_url`
+2. `animation_url`
+
+# Out of Scope
+
+Accessiblity of content is not covered by the standard.
+
+Properties other than resolvability of the token uri are out of scope. This implies that metadata might change over time (stability).
+
+# Metadata functions
+
+Some contracts have dedicated functions to provide metadata directly from the contract. This SIP does not define these functions.
+
+## Existing Examples
+
+Some early deployed contracts have metadata functions.
+
+### Boom
+
+The [NFT contract for Boom](https://explorer.stacks.co/txid/0x423d113e14791f5d60f5efbee19bbb05cf5e68d84bcec4e611f2c783b08d05aa?chain=mainnet) implements a variation of this trait using similar naming, but returning other types than response types.
+
+The function signatures for metadata are:
+
+- `(get-boom-meta () {uri: (string-ascii 35), name: (string-ascii 16), mime-type: (string-ascii 9)})` and
+- `(get-meta? uint {series-id: uint, number: uint, name: (string-utf8 80), uri: (string-ascii 2048), mime-type: (string-ascii 129), hash: (buff 64)})`
+
+### Badges
+
+The [badges contract](https://explorer.stacks.co/txid/0xb874ddbb4a602c22bb5647c7a2f8bfcafbbca7c0c663a175f2270ef3665f33de?chain=mainnet) defines metadata for nfts.
+
+The function signatures for metadata are:
+
+- `(get-badge-meta () {uri: (string-ascii 78111)})` and
+- `(get-meta? (uint) (optional {user: principal}))`
+
+# Backwards Compatibility
+
+This SIP defines meta data so that metadata for existing NFTs on other blockchains like Ethereum, Solana or WAX can be re-used for NFTs on the Stacks blockchain.
```

Comment:
"meta data" <-- did you mean "metadata"?

Reply:
fixed